.buildkite/hooks/pre-command

```diff
@@ -35,6 +35,12 @@ elif [[ "${OSTYPE}" == msys ]]; then
   mkdir -p /c/tmp/artifacts
 fi
 
-
 RAYCI_CHECKOUT_DIR="$(pwd)"
 export RAYCI_CHECKOUT_DIR
+
+if [[ "${NIGHTLY}" == "1" ]]; then
+  echo "Running nightly build"
+  export NIGHTLY_ALIAS=true
+else
+  export NIGHTLY_ALIAS=false
+fi
```

Comment:
this should not go to pre-command

Reply:
Where should be a better place for this? I was thinking of doing it straight on the command arg like `--nightly-alias $(if [[ "${NIGHTLY}" == "1" ]]; then echo "true"; else echo "false"; fi)` but it seems too long